src/org/usfirst/frc4904/robot/commands/ChangeTestingCounter.java

```diff
@@ -0,0 +1,42 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.humaninterface.operators.CheckOperator;
+import org.usfirst.frc4904.standard.LogKitten;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class ChangeTestingCounter extends Command {
+	int change;
+
+	public ChangeTestingCounter(int change) {
+		this.change = change;
+	}
+
+	@Override
+	protected void initialize() {}
+
+	@Override
+	protected void execute() {
+		LogKitten.wtf("Change " + change);
+		CheckOperator.counter += change;
+		if (CheckOperator.counter < 0) {
+			CheckOperator.counter = 0;
+		} else if (CheckOperator.counter > TestSubsystems.totalSubsystems + 1) {
+			CheckOperator.counter = TestSubsystems.totalSubsystems + 1;
+		}
```

Comment:
I think these should probably go in initialize to ensure that they only run once.